suse_migration_services/migration_config.py

```diff
@@ -42,11 +43,5 @@ def get_migration_product(self):
 
         The value returned is passed to the --product option in the
         zypper migration call
-
-        migration:
-          target:
-            product: product_name
         """
-        migration = self.config_data.get('migration')
-        if migration and migration.get('target'):
-            return migration.get('target').get('product')
+        return self.config_data.get('migration_product', 'SLES/15/x86_64')
```

Comment:
There is no default that we should hard code. The default product setup happens in the image description. Let's keep this code free from any static product string:

```python
return self.config_data.get('migration_product')
```

Reply:
Yes, I know. I plan to unify both files in our config file as agreed. I thought to do it in a different PR but more than happy to unify them here as well.

Reply:
Also, if no default here, dict.get defaults to ```None``` which would end up in the command ```zypper migration --flags....--product None```

Reply:
yes that's why the class MigrationConfig() exists in migration_config.py and in there the method get_migration_product(). I would adapt that method to the changed layout and raise an exception if no product is specified

Reply:
I'm not sure we need an exception, the setup of that product is optional and if no product is provided by the user, it should migrate to the default one in the image description.

Reply:
Got it. Ignored what I said. I will implement that.